File: Python Word examples/shapes/deleteShape.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.shapes_api.ShapesApi(configuration)

file_name = "ShapeTemplate.docx"
paragraph_path = "sections/0/paragraphs/0"
index = 1
extra = {'folder': 'input', 'dest_file_path': 'output/DeleteShape.docx'}
result = api.delete_shape(file_name, paragraph_path, index, **extra)

File: PHP Word Examples/ShapesApiDemo/deleteShape.php
<?php

use \Spire\Cloud\Word\Sdk\Client\Configuration;
use \Spire\Cloud\Word\Sdk\Client\Api\ShapesApi;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$fileName = "AddShapes.docx";
$folder = "input";
$storage = null;
$paragraph_path = 'Section/0/Body/0/Paragraph/0';
$index = 1;
$password = null;
$dest_file_path =  "Shapes/DeleteShape_1.docx";
$apiInstance->deleteShape($fileName, $paragraph_path, $index, $dest_file_path, $folder, $storage, $password);